pkg/env/env.go

```diff
@@ -9,10 +14,24 @@ type EnvInputs struct {
 	IronicAgentImage      string `envconfig:"IRONIC_AGENT_IMAGE"`
 	IronicAgentPullSecret string `envconfig:"IRONIC_AGENT_PULL_SECRET"`
 	IronicRAMDiskSSHKey   string `envconfig:"IRONIC_RAMDISK_SSH_KEY"`
+	RegistriesConfPath    string `envconfig:"REGISTRIES_CONF_PATH"`
 }
 
 func New() (*EnvInputs, error) {
 	env := &EnvInputs{}
 	err := envconfig.Process("", env)
 	return env, err
 }
+
+func (env *EnvInputs) RegistriesConf() (data []byte, err error) {
+	if env.RegistriesConfPath == "" {
+		return
```

Comment:
not a fan of these naked returns..

Reply:
Do we want to set an error here indicating that REGISTRIES_CONF_PATH is not set? It currently fails silently.

Reply:
I don't think we have to make it required (it doesn't fail, it just doesn't add a registry file to the ignition).